Obtaining a new token doesn't work because of error string mismatch

I'm using this lib to sync with my KIA in EU and receiving the following error:
```json
{"timestamp": "2026-04-23T19:55:00.376027Z", "level": "WARNING", "message": "Token expired detected, attempting refresh: Unknown error in API response: Key not authorized: token has expired", "module": "api_client", "function": "refresh_smart"}
```

My guess would be that this is because of a String mismatch in https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/blob/6a03d27a2f0664b4eed296b87c6672c61f0fa1af/hyundai_kia_connect_api/ApiImplType1.py#L82

The actual message is `token has expired` but the code says `Token is expired`. Maybe this was changed recently?

OK] Login successful!
[ERROR] OAuth error. Redirect URL: https://idpconnect-eu.kia.com/error?status=400&error=Bad+Request&error_description=Bad+Request.+It+was+classified+as+an+abusing+request+and+blocked.+client_id+%3A+fdc85c00-0a2f-4c64-bcb4-2cfb1500730a&loginUrl=&ui_locales=en-US
Cleaning up and closing the browser.
(.venv) PS C:\Users\PC\AppData\Local\Temp\KiaHyundaiToken>

voila ce que j'obtiens en retour, mais pas de token

--- a/hyundai_kia_connect_api/ApiImplType1.py
+++ b/hyundai_kia_connect_api/ApiImplType1.py
@@ -81,7 +81,8 @@ def _check_response_for_errors(response: dict) -> None:
     }
 
     error_message_to_exception_mapping = {
-        "Key not authorized: Token is expired": AuthenticationError
+        "Key not authorized: Token is expired": AuthenticationError,
+        "Key not authorized: token has expired": AuthenticationError,
     }
 
     if not any(

